End to end review of paper + what is a platform feedback (#309)
cncf/tag-app-delivery: platforms-whitepaper/v1alpha1/paper.md

make use of the platform to deliver product features at a higher pace, with 
reduced coordination."

The specific set of capabilities and scenarios supported
by a platform should be determined by the needs of your internal stakeholders. 
And while platforms _provide_ these required capabilities, it's critical to note 

> that platform teams should notalways _implement_ them themselves. Managed service 

Reviewer: ```suggestion
that platform teams should not always _implement_ them themselves. Managed service 
```

providers or dedicated internal teams can maintain backing implementations 
while platforms are the thinnest reasonable layer that provides consistency for these 
implementations that meets an organization's requirements. 
For example, a very simple "platform" could be a wiki page with links to 

> standard operating procedures to provisioncapabilities from providers, as described in [[3]].

Reviewer: ```suggestion
standard operating procedures to provision capabilities from providers, as described in [[3]].
```

make use of the platform to deliver product features at a higher pace, with 
reduced coordination."

The specific set of capabilities and scenarios supported
by a platform should be determined by the needs of your internal stakeholders. 

> And while platforms _provide_ theserequired capabilities, it's critical to note 

Reviewer: small typo: 
```suggestion
And while platforms _provide_ these required capabilities, it's critical to note 
```

Reviewer: It looks like something happened to the formatting of the text

> that platform teams should notalways _implement_ them themselves. Managed service 

Reviewer: ```suggestion
that platform teams should not always _implement_ them themselves. Managed service 
```

providers or dedicated internal teams can maintain backing implementations 

> while platforms are thethinnest reasonable layer that provides consistency for these 

Reviewer: ```suggestion
while platforms are the thinnest reasonable layer that provides consistency for these 
```

> implementationsthat meets an organization's requirements. 

Reviewer: ```suggestion
implementations that meet an organization's requirements. 
```

make use of the platform to deliver product features at a higher pace, with 
reduced coordination."

> The goal of any platform should be to drive organizational efficiency and 

Reviewer 2: IMO we shouldn't insist on organizational efficiency as the goal of a platform here. Some orgs I work with are concerned more about cost savings and security than productivity for example.

The first sentence of the next paragraph briefly highlights the idea that a platform should be driven by user requirements like productivity or others. Instead of adding a paragraph could we could update that first sentence of the next one for greater clarity as follows? Thanks!

"The specific set of capabilities and scenarios supported by a platform should be determined by the needs of users."

Because these platforms target no more and no less than an enterprise's internal
users we often refer to them as _internal_ platforms.

> Platforms are particularly relevant for cloud-native architectures because they

Reviewer 2: +1

like web application development or data analysis, aka MLOps.

Use cases an enterprise could meet with platforms might progress through the

> following [4]:

Reviewer 2: Let's not footnote this list, it's based on our original research.

1. Product developers can provision complete environments from templates
   combining run-time and development-time services required for specific
   scenarios, such as web development or MLOps.
1. Product developers and managers can observe functionality, performance, and cost

> 	of deployed services through automatuomatic instrumentation and single pane of glass

Reviewer: ```suggestion
	of deployed services through automatic instrumentation and single pane of glass
```

1. Product developers can provision complete environments from templates
   combining run-time and development-time services required for specific
   scenarios, such as web development or MLOps.

> 1. Budget holders get access to a dashboard for managing costs across business units with 

Reviewer 2: This list is to orient folks with example use cases so let's be a bit more general. Could you replace the current proposed item with the following? Thanks!

"Product developers and managers can observe functionality, performance and cost of deployed services through automatic instrumentation and standard dashboards."

[1]: https://www.atlassian.com/devops/frameworks/team-topologies
[2]: https://martinfowler.com/articles/talk-about-platforms.html
[3]: https://teamtopologies.com/key-concepts-content/what-is-a-thinnest-viable-platform-tvp

> [4]: https://thenewstack.io/architecture-and-design-considerations-for-platform-engineering-teams/

Reviewer 2: It's okay not to have a reference for this IMO, can we take it out or move to the bottom?

- Active users and resources: number of capabilities provisioned and used from the platform
- "Net Promoter Score" (NPS) or other surveys that measure user satisfaction with a product
- Metrics for developer productivity such as those discussed in the SPACE framework [[3]]

> - Monthly adoption rate and churn - measuring these two against each other can also help

Reviewer 2: The "Active users" item above captures this measure. If you feel deprovisioning is important to measure too could we modify the above item as follows? Thanks!

"Active users and resources: number of capabilities provisioned and used or deprovisioned via the platform"

Author: so on the metrics side - I believe active users is different than monthly adoption rate. active users measures daily engagement and that can be amongst your current users

monthly adoption rate and churn is meant for adoption - how many customers are you gaining and losing
